<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Thomas Gorochowski" />
  <meta name="dcterms.date" content="2025-11-19" />
  <title>AIEBaB-2026</title>
  <link rel="preconnect" href="https://fonts.googleapis.com">
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
  <link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:ital,wght@0,100..800;1,100..800&display=swap" rel="stylesheet">
  <link rel="stylesheet" href="reset.css" />
  <link rel="stylesheet" href="index.css" />
</head>
<body>
<table class="header">
  <tr>
    <td colspan="2" class="width-auto">
      <h1 class="title">AI, Engineering Biology and Beyond 2026</h1>
      <span class="subtitle">15-16 January 2026, Bristol, UK</span>
    </td>
    <th>Version</th>
    <td class="width-min"><time style="white-space: pre;">v2.0.0</time></td>
  </tr>
</table>

<nav align="left">
    <button style="padding-top:5px; padding-bottom:5px;"><a href="https://aiebab.github.io" style="text-decoration: none;">Home</a></button> <button style="padding-top:5px; padding-bottom:5px;"><a href="https://aiebab.github.io/schedule.html" style="text-decoration: none;">Schedule</a></button> <button style="padding-top:5px; padding-bottom:5px;"><a href="https://aiebab.github.io/exhibits.html" style="text-decoration: none;">Exhibits</a></button>
</nav>

<p>Goto:
<ul style="margin-top: -16px;">
    <li><a href="#talks-keynotes-spotlight">Keynotes and Spotlights</a></li>
    <li><a href="#talks-contributed">Contributed Talks</a></li>
    <li><a href="#talks-translation-impact">Translation and Impact</a></li>
    <li><a href="#posters">Posters</a></li>
</ul>
</p>

<h1 id="talks">Talks</h1>

<h2 id="talks-keynotes-spotlight">Keynotes and Spotlights</h2>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Toward AI-driven genetic circuit design</h3>
<p style="margin-top: 6px; margin-left: 16px;">Caleb Bashor (Rice University, USA)</p>
</summary>
<p style="margin-left: 16px;">Engineering synthetic regulatory circuits with precise input-output behavior—a central goal in synthetic biology—remains encumbered by the inherent molecular complexity of cells. Non-linear, high-dimensional interactions between genetic parts and host cell machinery make it difficult to design circuits using first principles biophysical models.  Adopting data-driven approaches that integrate modern machine learning (ML) tools and high-throughput experimental approaches into the synthetic biology design/build/test/learn process could dramatically accelerate the pace and scope of circuit design, yielding workflows that rapidly and systematically discern design principles and achieve quantitatively precise behavior. I will argue that the application of ML to circuit design can occur at three distinct scales, and describe work in my group making progress in each of them: 1) learning relationships between part sequence and function; 2) determining how part composition determines circuit behavior; 3) understanding how function varies with genomic/host-cell context. The work points toward a future where ML-driven genetic design is used to program robust solutions to complex problems across diverse biotechnology domains.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Generative and Multimodal AI for Protein Design and Interaction Screening</h3>
<p style="margin-top: 6px; margin-left: 16px;">Haiping Lu (University of Sheffield, UK)</p>
</summary>
<p style="margin-left: 16px;">Generative AI is opening new possibilities for protein design, while interaction screening remains essential for prioritising candidates in engineering biology workflows. In this talk, I will present generative and multimodal AI methods for protein sequence design and protein–ligand interaction screening. I will highlight a diffusion-based approach for structure-guided protein sequence design and an interpretable model for protein–small-molecule interaction screening. Together, these examples illustrate how multimodal representations, rigorous evaluation, benchmarking, and deployment-aware design are critical for translating AI advances into reliable design–screen pipelines for engineering biology.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">TBC</h3>
<p style="margin-top: 6px; margin-left: 16px;">Henry Lee (Cultivarium, USA)</p>
</summary>
<p style="margin-left: 16px;">TBC.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Synthetic Biology 2.0: the Dawn of a New Era</h3>
<p style="margin-top: 6px; margin-left: 16px;">Huimin Zhao (University of Illinois at Urbana-Champaign, USA)</p>
</summary>
<p style="margin-left: 16px;">Synthetic biology aims to design novel or improved biological systems using engineering principles, which has broad applications in medical, chemical, food, and agricultural industries. Thanks to rapid advances in DNA sequencing and synthesis, genome editing, artificial intelligence/machine learning (AI/ML), and laboratory automation, synthetic biology has entered a new phase of exponential growth. In this talk, I will highlight our recent work on the development of a self-driving biofoundry, AI/ML tools, non-model organisms, and novel enzymes for synthetic biology applications. Examples include but are not limited to: (a) BioAutomata: an AI-powered self-driving biofoundry for protein engineering, pathway engineering, and metabolic engineering in organisms ranging from bacteria to yeast to plant cells, (b) CLEAN: an AI tool for enzyme function prediction, (c) EZSpecificity: an AI tool for enzyme substrate specificity prediction, (d) engineered low-pH tolerant yeasts for cost-effective production of organic acids, and (e) repurposed enzymes with new-to-nature reactivity for asymmetric synthesis.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Molecular Machine Learning for Sustainable Molecular Designs</h3>
<p style="margin-top: 6px; margin-left: 16px;">Jana M. Weber (TU Delft, The Netherlands)</p>
</summary>
<p style="margin-left: 16px;">Designing sustainable molecules requires navigating vast design spaces with sparse information. In this talk, I will present key building blocks of molecular machine learning and show how they enable tailored exploration of chemical space. I will illustrate these concepts through two case studies. The first focuses on the design of photocatalytic polymers, highlighting challenges in polymer representation, predictive modelling with limited data, and inverse design strategies for discovering promising candidates. The second examines what is needed for reaction system evaluation and optimisation of these in vast chemical space.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Cell-machine interfaces for control and learning</h3>
<p style="margin-top: 6px; margin-left: 16px;">Jean-Baptiste Lugagne (University of Oxford, UK)</p>
</summary>
<p style="margin-left: 16px;">Engineering biology increasingly depends on the integration of data, models, and automation. Advances at the intersection of synthetic biology, control theory, and machine learning have led to the development of cell-machine interfaces: experimental platforms that directly couple living cells to computers through microfluidics, automated microscopy, and optogenetics. These cybergenetic systems establish a foundation for real-time learning and closed-loop control of cellular behaviour. In the first part of this talk, I will present how such interfaces have been used to dissect and manipulate antibiotic-resistance dynamics in bacteria. By combining data-driven control algorithms, high-throughput imaging, and single-cell optogenetics, we were able to precisely regulate gene expression in thousands of individual cells and generate rich datasets that reveal the regulatory principles underpinning tolerance and resistance acquisition. I will then describe how my group is extending this strategy to new biological systems and new application domains. In particular, I will present how we plan to use such platforms to emulate real-world, time-varying conditions in high throughput and at the microfluidic scale for training and validating predictive models, notably for biomanufacturing and scale-up. Finally, I will discuss how these interfaces create a bridge between biological experimentation and advanced AI/ML methods, and support autonomous experimentation and optimisation of biological systems. This work aims to establish experimental-computational infrastructures that enable AI to directly interact with, learn from, and ultimately design biological systems across scales.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Open-Ended, Quality Diversity, and AI-Generating Algorithms in the Era of Foundation Models</h3>
<p style="margin-top: 6px; margin-left: 16px;">Jeff Clune (University of British Columbia, Canada)</p>
</summary>
<p style="margin-left: 16px;">Foundation models (e.g. large language models) create exciting new opportunities in our longstanding quests to produce open-ended and AI-generating algorithms, wherein agents can truly keep innovating and learning forever. In this talk, I will introduce quality diversity, open-ended, and AI-generating algorithms and share some of our recent work harnessing the power of foundation models to unleash their potential. I will cover our recent work including OMNI (Open-endedness via Models of human Notions of Interestingness), Video Pre-Training (VPT), Automatically Designing Agentic Systems (ADAS), the Darwin Gödel Machine, and The AI Scientist.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">A Slice of Infinity: Building Robust, Interpretable AI for Open-ended Biodesign</h3>
<p style="margin-top: 6px; margin-left: 16px;">Michiel Stock (Ghent University, Belgium)</p>
</summary>
<p style="margin-left: 16px;">The goal of synthetic biology is to design new-to-nature components, pathways, and organisms, often by combining, adapting, or changing existing biological parts in creative ways. Any computational tools to aid the biodesign process must be flexible, composable, and adaptable, as biology itself is. Hyperdimensional computing (HDC) is a promising brain-inspired learning paradigm that leverages the peculiar properties of high-dimensional spaces. In this talk, we explore how HDC can be used to efficiently learn from multi-modal, structured data and generate interpretable, robust models for biodesign.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Bottlenecks to Biological Progress</h3>
<p style="margin-top: 6px; margin-left: 16px;">Niko McCarty (Asimov Press, USA)</p>
</summary>
<p style="margin-left: 16px;">Our ability to predictably engineer biology will greatly improve in the 21st century, aided in part by AI. These advances could prove useful in creating a more flourishing future; not only for medicine, but also for animal welfare, terraforming planets, and so on. In this talk, I discuss two classes of bottlenecks that I think are limiting biology progress, explain how they are being solved today, and discuss the good outcomes which bioengineers can help build as a result.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">TBC</h3>
<p style="margin-top: 6px; margin-left: 16px;">Sabine Hauert (University of Bristol, UK)</p>
</summary>
<p style="margin-left: 16px;">TBC.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">RNA strand exchange circuits as a general-purpose molecular programming language for synthetic biology</h3>
<p style="margin-top: 6px; margin-left: 16px;">Samuel Schaffter (National Institute of Standards and Technology, USA)</p>
</summary>
<p style="margin-left: 16px;">A major goal of synthetic biology is to program biological systems with the same precision with which we program electronic devices, ultimately enabling the next generation of diagnostics, therapeutics, and biotechnologies. Great strides have been made, but a general challenge is most molecular programming paradigms are developed to operate in a narrow set of environments or applications. Further, sensing, information processing, and signal transduction are often coupled within a single device or design, making interoperability difficult. In contrast, in electronic computing inputs are converted into a universal machine language for information processing, and outputs of the machine instructions are mapped back to an application specific response. This enables seamless communication and integration across devices and applications. To enable similarly broad programmability of biological systems, we are developing a molecular equivalent of these machine instructions based on genetically encoded RNA strand exchange circuits called ctRSD circuits. RNA strand exchange reactions are easily programmed via predictable base pairing interactions universal across cell types, enabling wide operability. Further, in vitro nucleic acid strand exchange reactions are the most scalable and programmable biomolecular systems to date, demonstrating, for example, neural network-based classification of hundreds of inputs. I will provide an overview of our work operating ctRSD circuits across different environments and connecting these RNA circuits to different classes of inputs (metabolites and proteins) and outputs (proteins). Lastly, I will discuss the additional measurements needed to fully characterize circuit performance and properly inform AI/ML models that could ultimately automate the molecular circuit design process.</p>
</details>

<h2 id="talks-contributed">Contributed Talks</h2>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Deep mutational learning for effective enzyme engineering</h3>
<p style="margin-top: 6px; margin-left: 16px;">Alperen Dalkiran (University of Edinburgh, UK)</p>
</summary>
<p style="margin-left: 16px;">Deep learning is transforming our ability to engineer proteins by identifying the complex rules that map genotype to phenotype, reducing costly rounds of trial-and-error experimentation. In this talk, I will present our deep mutational learning system designed to accelerate enzyme optimization. We first learned a high-quality fitness landscape of peroxidase activity of human myoglobin, using a yeast display assay of more than 6,000 variants, protein language models and supervised neural networks. Using the pretrained model, we computationally screened more than 4M unseen double mutants, and selected 20 of the highest-scoring myoglobin variants for experimental validation. This approach led to a 100% validation success rate, with every tested variant showing improved peroxidase function compared to the wild type. Our best variant displayed a nearly 5-fold improvement in catalytic efficiency. Furthermore, we validated the superior performance of the three top variants in soluble form. Our work presents a robust and scalable framework for data-driven enzyme engineering. It demonstrates how protein language models can be utilized to guide the discovery of high-performance biocatalysts, which can then be leveraged to construct novel metabolic pathways. I will discuss current extensions of these technologies to design novel gene regulatory elements in cyanobacteria for environmental applications as part of the CYBER Mission Award in Engineering Biology.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Generative design and construction of functional plasmids with a DNA language model</h3>
<p style="margin-top: 6px; margin-left: 16px;">Angus Cunningham (University College London, UK)</p>
</summary>
<p style="margin-left: 16px;">DNA language models are emerging as a new approach for biological sequence generation, but their practical capabilities remain unclear. In this work, we explore whether such models can move beyond producing plausible DNA sequences to generating plasmids that are experimentally testable. We focus on Escherichia coli plasmids as a tractable system and present an end-to-end workflow spanning large-scale plasmid data curation, model fine-tuning, sequence generation, and bioinformatic screening. Using a GPT-style DNA language model, we generate synthetic plasmid backbones under both minimal and function-conditioned prompts, followed by stringent filtering to identify viable candidates. Selected designs were synthesised and evaluated in vivo, assessing growth, antibiotic resistance, and reporter expression. This work aims to probe the current limits of DNA language models and highlight both their potential and constraints as tools for generative biological design.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Computational inference of complex feedstock composition</h3>
<p style="margin-top: 6px; margin-left: 16px;">Charlotte Merzbacher (Differential Biology GmBH, Germany)</p>
</summary>
<p style="margin-left: 16px;">The rise of antibiotic resistance poses a significant global health threat, necessitating innovative solutions like phage therapy. Phages, viruses that target specific bacteria, offer a promising alternative due to their inexhaustible evolution potential, specificity and synergy with most antibiotics. However, two main challenges hinder their efficacy: the time-intensive process of identifying suitable phages for infections and the rapid development of bacterial resistance. My project addresses these challenges through leveraging the advances in synthetic biology and an evolution-aware machine learning framework. The latter incorporates protein language models, phage-host interaction classifiers and evolutionary algorithms to design improved receptor-binding proteins, which are key determinants of phage infectivity and host range. The resulting databases and models aid in the transition from the current empirical phage screening methods toward knowledge-driven bio design of engineered phages with improved and adaptable antibacterial activity.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Reprogramming Development through Morphogenetic Perturbations</h3>
<p style="margin-top: 6px; margin-left: 16px;">Elias Najarro (IT University of Copenhagen, Denmark)</p>
</summary>
<p style="margin-left: 16px;">Developmental systems rely on a variety of mechanisms to robustly produce their target morphologies. One such mechanism is morphogenetic gradients: non-uniform concentrations of biomolecules that scaffold patterning. We first present a computational model showing how morphogenetic gradients enable symmetry breaking in isotropic, purely local developmental programs. We then demonstrate how morphogenetic fields can be hijacked to steer development toward novel patterns through targeted perturbations—without modifying the developmental rules themselves.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Iterative NAND Hybrid Riboswitch Design by Deep Batch Bayesian Optimization</h3>
<p style="margin-top: 6px; margin-left: 16px;">Erik Kubaczka (TU Darmstadt, Germany)</p>
</summary>
<p style="margin-left: 16px;">Designing large genetic circuits requires genetic regulatory devices that perform complex logic operations without placing an excessive metabolic burden on the host. Hybrid riboswitches are synthetically enhanced, compact RNA elements (less than 100 nucleotides) that form a tertiary structure and can specifically bind multiple different target molecules. They can be used to design genetic regulators that emulate Boolean logic. However, their functionality is sensitive to even single-nucleotide sequence changes, posing a challenge to targeted engineering. This study's goal is to design 2-input hybrid riboswitches that emulate Boolean NAND logic in yeast. We therefore developed a novel sequence design framework that combines in vivo screenings with uncertainty-aware batch Bayesian optimization, enabling score-driven sequence design. Through an initial screening, we discovered a hybrid riboswitch exhibiting NAND behavior. We then further engineered the riboswitch sequence by iteratively refining it in a design-build-test-learn (DBTL) cycle. Core to this is Bayesian optimization with an ensemble sequence-to-function surrogate trained on in vivo characterization data and pre-trained on custom riboswitch sequencing data. Ultimately, the hybrid riboswitch exhibits near digital NAND behavior. With its focus on model-based and score-driven design, our proposed method can complement experiment-driven approaches by enabling fine-grained adaptation of functionality, including constructs sensitive to single-nucleotide changes.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Development of an AI-Native Open Data Lakehouse for Automated Synthetic Biology</h3>
<p style="margin-top: 6px; margin-left: 16px;">Matt Burridge (Newcastle University, UK)</p>
</summary>
<p style="margin-left: 16px;">Synthetic Biology is a highly interdisciplinary field that generates both computational and physical experimental data. To truly understand the result of an experiment, you must integrate information from every stage of an experimental workflow. Advances in lab automation and AI-driven experimental design have increased both the scale and complexity of these workflows. Biofoundries provide integrated infrastructure to handle this, however many laboratories rely on isolated instruments and software systems that operate as disconnected silos. These systems often produce heterogeneous data formats with limited structured metadata, hindering data integration, reproducibility, and downstream computational analysis. To address this, we present a data architecture that combines RO-Crates, domain ontologies and a vendor-agnostic data lakehouse paradigm. We demonstrate this architecture through a semi-automated enzyme characterisation use case, ingesting data from a metagenomic enzyme discovery workflow, spanning computational pipelines and automated laboratory equipment. Within the lab, we developed a lightweight automation data hub for RO-Crate ingestion, a Rust-based library (ro-crate-rs), and a labware barcoding system to capture provenance during both automated and manual work that is backed by PROV-O based data models. This open-source, vendor-independent approach enables robust, trustworthy, machine-readable wet-lab datasets that can be integrated with computational data to support future engineering biology workflows.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Protein design enables highly accurate sequence-to-expression models for an integral membrane protein</h3>
<p style="margin-top: 6px; margin-left: 16px;">Paul Curnow (University of Bristol, UK)</p>
</summary>
<p style="margin-left: 16px;">The recombinant expression of integral membrane proteins is a major bottleneck across the biosciences. This includes industrial and academic research in bioengineering and synthetic biology. Computational sequence-to-expression models would be potentially transformative in addressing this challenge but the data needed to build such models is absent. Here, we describe a novel approach to generating ML-ready expression data that exploits membrane protein libraries derived from computational sequence design. We determine the recombinant expression phenotype of this library in E. coli, obtain gene sequences exhibiting high or low expression, and use this new dataset to develop very accurate predictive classifiers. We find that these sequence data are compatible with different embeddings and ML architectures, including explainable models, and use feature analysis to reveal that particular amino acid residues exert a strong influence on recombinant protein production. This residue-level insight is validated experimentally via model-guided engineering of membrane protein expression levels. While the results presented here are specific to our training data and conditions, we propose that similar methods could be applied to other membrane proteins to derive a general model of the expression fitness landscape in common recombinant chassis.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">Upscaling Bacteriophage Therapy: AI and synthetic biology driven framework</h3>
<p style="margin-top: 6px; margin-left: 16px;">Victor Németh (Ghent University, Belgium)</p>
</summary>
<p style="margin-left: 16px;">The rise of antibiotic resistance poses a significant global health threat, necessitating innovative solutions like phage therapy. Phages, viruses that target specific bacteria, offer a promising alternative due to their inexhaustible evolution potential, specificity and synergy with most antibiotics. However, two main challenges hinder their efficacy: the time-intensive process of identifying suitable phages for infections and the rapid development of bacterial resistance. My project addresses these challenges through leveraging the advances in synthetic biology and an evolution-aware machine learning framework. The latter incorporates protein language models, phage-host interaction classifiers and evolutionary algorithms to design improved receptor-binding proteins, which are key determinants of phage infectivity and host range. The resulting databases and models aid in the transition from the current empirical phage screening methods toward knowledge-driven bio design of engineered phages with improved and adaptable antibacterial activity.</p>
</details>

<h2 id="talks-translation-impact">Translation and Impact</h2>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">FateView: AI-Powered, non-destructive live cell analytics and cell fate forecasting</h3>
<p style="margin-top: 6px; margin-left: 16px;">Christopher Gribben (CellVoyant, UK)</p>
</summary>
<p style="margin-left: 16px;">A major bottleneck in engineering biology and cell manufacturing is that critical quality attributes (e.g., identity, state, and trajectory) are often assessed using destructive, end-point assays. This creates long feedback loops, limits the ability to intervene during culture, and increases the risk of batch failure during scale-up. Here, we present FateView, an AI-native approach for non-destructive, label-free monitoring of cell fate decisions during differentiation and manufacturing. FateView combines longitudinal in-process measurements (e.g., brightfield imaging and process metadata) with machine-learning models that first detect and segment cells, then extract post-detection morphological and dynamic features over time to predict downstream outcomes ahead of traditional QC. This approach allows earlier decisions that improve consistency, reduce cost, and support GMP-aligned manufacturing workflows.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">TBC</h3>
<p style="margin-top: 6px; margin-left: 16px;">Sarah Jackson (Twist Bioscience, USA)</p>
</summary>
<p style="margin-left: 16px;">TBC.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">AI, Engineering Biology and the BBSRC - a funder’s perspective</h3>
<p style="margin-top: 6px; margin-left: 16px;">Daniela Hensen (UKRI-BBSRC, UK)</p>
</summary>
<p style="margin-left: 16px;">The presentation will outline BBSRC’s approach towards support for AI, engineering biology and their convergence. We will share high level insights from our current portfolio of activities and outline our thinking for the future of the field.</p>
</details>

<details>
<summary>
<h3 style="margin-top: -16px; margin-left: 16px; margin-bottom: 0px;">TBC</h3>
<p style="margin-top: 6px; margin-left: 16px;">Kerstin Kinkelin & Kathleen Sedgley (University of Bristol, UK)</p>
</summary>
<p style="margin-left: 16px;">TBC.</p>
</details>

<h1 id="posters">Posters</h1>

<ol style="padding-bottom:10px;">
<li><p style="margin-top: -16px; margin-left: 35px;"><span style="font-weight: var(--font-weight-bold);">Testing reversible disassembly of ferritin nanocages for cargo encapsulation</span><br>Eli Lewis (University of Bristol, UK)</p></li>

<li><p style="margin-top: -16px; margin-left: 35px;"><span style="font-weight: var(--font-weight-bold);">Testing reversible disassembly of ferritin nanocages for cargo encapsulation</span><br>Eli Lewis (University of Bristol, UK)</p></li>
</ol>

</body>
</html>
